tests/unit/shared/lib/fs-test.js

```diff
@@ -0,0 +1,224 @@
+// /* eslint-disable */
+const imports = require('esm')(module);
+const { join } = require('path');
+const mockFs = require('mock-fs');
+const test = require('tape');
+
+const fs = imports(join(process.cwd(), 'src', 'shared', 'lib', 'fs.js'));
+
+test('Module exists', t => {
+  t.plan(1);
+  t.ok(fs, 'fs exists');
+});
+
+test('fs.exists', async t => {
+  const tests = ['test/somefile', 'test/asubdirectory/somefile', 'somefile'];
+  t.plan(tests.length * 3);
```

Comment:
I see having to plan tests as being a big source of headaches for someone not familiar with this library. Is there a way to avoid having to specify the number of tests? Or could we document this?

Reply:
Can document. Will explain why you should plan your tests in comments above.